integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -221,7 +221,7 @@ def auto_complete_address(request, region_slug):
     return JsonResponse(
         data={
             "postcode": address.get("postcode"),
-            "city": address.get("city"),
+            "city": address.get("city") or address.get("town"),
```

Comment:
```suggestion
            "city": address.get("city")
            or address.get("town")
            or address.get("village"),
```
I noticed the response can also contain `village`, e.g. for:
[Hauptstraße 85a
35716 Ewersbach](http://nominatim.maps.tuerantuer.org/nominatim/search?street=Hauptstra%C3%9Fe%2085a&postalcode=35716&city=Ewersbach&addressdetails=1)